public/scripts/expandMenuManager.js

```diff
@@ -1,17 +1,25 @@
-const renderPopout = (e) => {
```

Comment:
not sure why you rewrote this as I had the popout function working perfectly and already coded